src/main/java/org/eclipsefoundation/adopters/model/Adopter.java

```diff
@@ -0,0 +1,96 @@
+package org.eclipsefoundation.adopters.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.google.gson.annotations.SerializedName;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```

Comment:
Is it necessary as you already customize the `ObjectMapper` in `CustomJacksonConfig`